hc/api/models.py

```diff
@@ -112,13 +111,10 @@ def prune_pings(self, keep_limit):
         if len(cutoff) == 0:
             return 0
 
-        cutoff_date = cutoff[0].created
-        q = Ping.objects.filter(owner=self, created__lte=cutoff_date)
+        cutoff_id = cutoff[0].id
```

Comment:
using `created` a timestamp is not safe, two checks may have the exact same `created` time, but `id` is guaranteed to be unique and monotonically increasing